spec/game_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper'
+
+describe "game" do
+
+  describe "Guess!" do
+    before :each do
+      dictionary = Dictionary.new %w(apple banana cheese danish egg fry)
+      @game = Cinch::Plugins::Game.new(dictionary,dictionary, solution: "banana")
+    end
+
+    it "can tell when you guess too high" do
+      expect(@game.guess!("apple")).to eql(:missed_north)
+    end
+
+    it "can tell when you guess too low" do
+      expect(@game.guess!("cheese")).to eql(:missed_south)
+    end
+
+    it "can tell when you guess something not a word" do
+      expect(@game.guess!("thuslymuchly")).to eql(:not_a_word)
```

Comment:
hm, might want to refactor the :not_a_word symbol to `:not_in_dict` or something similar .. I'm not sure if more thought needs to go into it considering the differences between presence in ref/gameplay dict